subjects/drop_the_blog/README.md

```diff
@@ -0,0 +1,105 @@
+## drop_the_blog
+
+### Instructions
+
+Define the following structures:
+
+- `Blog`: containing:
+  - `drops`: that will save the number of dropped articles.
+  - `states`: that will save the state of multiple articles. If the article is not dropped, the state will be `false`, otherwise it will be `true`.
+- `Article`: containing:
+  - `id` as `usize`.
+  - `body`: as `String`.
+  - `parent`: a link to the structure `Blog`. (Tip: this should be a reference).
+
+You'll need to also add the following associated functions to the structures:
+
+- `Blog`:
```

Comment:
Missing a new line here to be `prettier` complaint

Reply:
done. Also added extra empty lines and removed `:` at the start of elements' list for more consistency.